### PR TITLE
Fixes for IWD Arcane & Divine Spell Pack compatibility with SR installation

### PR DESCRIPTION
**Pull request by AshLancer**
Hello,

SR-relative settings code was checking for the wrong component, which meant that there were in fact NO checks being made for a Spell Revisions installation. SCS's tpa files do not have this issue.

The other problem is that the skip_sr.2da file was incomplete, and in fact only contained the bare minimum list of spells to skip when a SR installation is present. I went through and cross-referenced every single spell between IWDification and SR (there aren't that many), and added spells that were already present to the exclusion list. Symbol, Hopelessness and Symbol, Pain are so functionally similar to Symbol of Stunning and Symbol of Weakness, that I see no purpose to installing both. The only other exclusion I made was Stalker, since it's nearly the same as SR's Summon Shambling Mound.

In short: IWDification was just blatantly overriding SR spells, and in some cases even copying over duplicate spells, without ever notifying the user.

### CODE REVIEW

**Comment by GraionDilach** [comment]
> Symbol, Hopelessness and Symbol, Pain are so functionally similar to Symbol of Stunning and Symbol of Weakness, that I see no purpose to installing both. The only other exclusion I made was Stalker, since it's nearly the same as SR's Summon Shambling Mound.

Unless they share the IDS identifier, you can't do this. With v34, SCS defaults to using the IWD spell list for SR+IWD installs, and Smarter Mages/Clerics will throw Not_found warnings if an IWD spell is missing from SPELL.IDS.

**Comment by AshLancer** [comment]
> > Symbol, Hopelessness and Symbol, Pain are so functionally similar to Symbol of Stunning and Symbol of Weakness, that I see no purpose to installing both. The only other exclusion I made was Stalker, since it's nearly the same as SR's Summon Shambling Mound.
> 
> Unless they share the IDS identifier, you can't do this. With v34, SCS defaults to using the IWD spell list for SR+IWD installs, and Smarter Mages/Clerics will throw Not_found warnings if an IWD spell is missing from SPELL.IDS.

Sorry, you are correct. In that case, let's simply allow the symbol spells to install. Everything else should be present in the SR SPELL.IDS. I don't think that SCS uses CLERIC_CAUSE_SERIOUS_WOUNDS_IWD or CLERIC_CAUSE_CRITICAL_WOUNDS_IWD, but someone can double-check that.

**Comment by AshLancer** [comment]
Want to close this, as it's not working the way it should. Needs further investigation. Thank you.
